backdrop_upgrade_status.core.inc

```diff
@@ -246,6 +246,15 @@ function _backdrop_upgrade_status_backdrop_core(&$data, $project) {
       $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');
       break;
 
+    case 'entityreference':
+      $data[$project]['in_core_since'] = '1.x';
+      $data[$project]['in_core_note'] = t('Moved into core in v1.23.');
+      $data[$project]['in_core_warning'] = t('See <a href="@entityreference">the documentation</a> for more information.', array(
```

Comment:
We should remove this warning, and instead add
```
      $data[$project]['in_core_complete'] = TRUE;
      $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');

```

Reply:
@jenlampton they're the same functionality.

Reply:
@jenlampton I was following the pattern like this one:

```
case 'entitycache':
      $data[$project]['in_core_since'] = '1.x';
      $data[$project]['in_core_note'] = t('Integrated into core Entity module in v1.16.');
      $data[$project]['in_core_warning'] = t('See <a href="@entitycache">the documentation</a> for more information.', array(
        '@entitycache' => 'https://docs.backdropcms.org/change-records/entity-caching-added-to-core',
      ));
      $data[$project]['in_core_upgrade_status'] = t('Leave enabled.');
```

If we remove the warning from one, why not the others?

Reply:
@herbdool @jenlampton 
Can we split the difference (and open a new issue on whether to remove the warning on these types of cases)? Maybe inserting `Leave enabled.` in front of the link to the more detailed change record info?

Reply:
We don't need a warning if there's no differences that could introduce a risk. Entity cache had a rewrite introducing some potentially dangerous differences, hence the warning. 

We can definitely open an issue to more clearly discuss and define when a warning is necessary. My original thought was "only when people need to be warned" about the change. We could certainly add some other kind of notice with "extra information" that isn't a warning :)